apps/services/user-profile/src/app/v2/userToken.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { ApiSecurity, ApiTags } from '@nestjs/swagger'
+import { Controller, Get, UseGuards, Headers } from '@nestjs/common'
+
+import { IdsAuthGuard, Scopes, ScopesGuard } from '@island.is/auth-nest-tools'
+import { UserProfileScope } from '@island.is/auth/scopes'
+import { Audit } from '@island.is/nest/audit'
+import { Documentation } from '@island.is/nest/swagger'
+
+import { UserTokenService } from './userToken.service'
+import { UserDeviceTokenDto } from '../user-profile/dto/userDeviceToken.dto'
+
+const namespace = '@island.is/user-profile/v2/userTokens'
+
+@UseGuards(IdsAuthGuard, ScopesGuard)
+@Scopes(UserProfileScope.admin)
+@ApiTags('v2/userTokens')
+@ApiSecurity('oauth2', [UserProfileScope.admin])
+@Controller({
+  path: 'userTokens',
+  version: ['2'],
+})
+@Audit({ namespace })
+export class UserTokenController {
+  constructor(private readonly userTokenService: UserTokenService) {}
+
+  @Get('/.nationalId')
```

Comment:
Then this is handling the base route of the controller
```suggestion
  @Get()
```